packages/lit-dev-content/samples/examples/react-slots/app.tsx

```diff
@@ -0,0 +1,14 @@
+import {createComponent} from '@lit-labs/react';
+import {React} from "./faux-react.js";
+import {LyricalSlots as LyricalSlotsWC} from './lyrical-slots.js';
+
+const LyricalSlots = createComponent(React, 'lyrical-slots', LyricalSlotsWC);
+
+export const App = () => (
+  <>
+    <LyricalSlots>
+      <div slot="clowns">&#129313; &#129313; &#129313;</div>
```

Comment:
This example isn't rendering anything for me (nor would I expect it to, because neither of these DIVs matches ones of the component's named slots). 

I don't know if that was the intention, but it seems pretty confusing if so.

Reply:
this should be resolved now, my bad arthur